ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -1,3 +1,36 @@
+if get_option('enable-nnfw-runtime')
+  filter_sub_nnfw_sources = ['tensor_filter_nnfw.c']
+
+  nnstreamer_filter_nnfw_sources = []
+  foreach s : filter_sub_nnfw_sources
+    nnstreamer_filter_nnfw_sources += join_paths(meson.current_source_dir(), s)
+  endforeach
+
+  nnfw_dep = dependency('nnfw', required: false)
```

Comment:
required: true?

Reply:
No. nnfw is not producing a pkg-config file; thus we cannot use mseon's dependency, yet.

Reply:
I thought given `enable_nnfw` will false remain till this package has all the fixes